JamesRobertSutcliffe/personality-music-recommender
# Update docs with Recommendation System API

Closes #30

---

This is a simple PR that cleans up the repository. I updated the docs and removed ml-related files, as these are now hosted on [Hugging Face](https://huggingface.co/spaces/patdel0/personality-music-recommender/tree/main).

<img width="1055" alt="image" src="https://github.com/JamesRobertSutcliffe/personality-music-recommender/assets/53922624/12845ef2-41ee-4138-b62d-be1ebd741ed9">


## Testing
1. Add this code anywhere in the application: 
```js
fetch('https://patdel0-personality-music-recommender.hf.space/recommend?mbti_type=entj&num_of_songs=5')
  .then(data => data.json())
  .then(result => console.log(result))
```
2. Confirm the fetch request returns the recommended songs objects
